server/config/config.go

```diff
@@ -654,6 +668,38 @@ func (m *MDMConfig) loadAppleBMEncryptedToken() ([]byte, error) {
 	return tokBytes, nil
 }
 
+// MicrosoftWSTEP returns the parsed and validated TLS certificate for Microsoft WSTEP.
+// It parses and validates it if it hasn't been done yet.
+func (m *MDMConfig) MicrosoftWSTEP() (cert *tls.Certificate, pemCert, pemKey []byte, err error) {
+	// TODO: should we also implement support for setting raw bytes in the config (like we do for Apple MDM)?
```

Comment:
This TODO will be addressed before the PR is finalized.

Reply:
Returning raw bytes is not required for now